app/front/scripts/services/download-package.js

```diff
@@ -80,8 +80,8 @@
               StorageService.clearApplicationState()
                   .then(function() {
                     var packageName = PackageService.getAttributes().name;
-                    var owner = LoginService.email;
-                    state.packagePublicUrl = '/viewer/'+owner+':'+packageName; //dataPackage.uploadUrl;
+                    var owner = LoginService.userid;
```

Comment:
So the real question is what defines a 'user': is it 
1. the unique ID coming from the provider - which will make Facebook's Paul and Google's Paul two different entities, even when using the same email?
2. The email, which will make Paul@okfn and Paul@gmail two different entities, and Paul@guthub will match one of them, based on the email you have registered there.

Both are valid options, the question is which is less confusing to users.